orb.yml

```diff
@@ -125,16 +129,9 @@ commands:
               curl  -XPOST -s -H 'Content-Type: application/json' \
                 -H "Authorization: Bearer $<<parameters.api_key>>" --data \
                 "{
-                  \"version\": \"<<parameters.version>>\",
-                  \"url\":\"$CIRCLE_BUILD_URL\",
-                  \"refs\":[{
-                    \"repository\":\"$CIRCLE_PROJECT_REPONAME\",
-                    \"commit\":\"$CIRCLE_SHA1\"
-                  }],
-                  \"projects\":[
-                    \"$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME\"
-                  ]
-                }" "https://sentry.io/api/0/organizations/<<parameters.organization>>/releases/"
+                  \"environment\":\"$<<parameters.environment>>\",
+                  \"url\":\"$CIRCLE_BUILD_URL\"
+                }" "https://sentry.io/api/0/organizations/<<parameters.organization>>/releases/<<parameters.tag>>/deploys/"
```

Comment:
@spacemup we are referencing `parameters.tag` but it does not exist

Reply:
ah oops

Reply:
https://github.com/titel-media/gcp-gke-orb/pull/4/commits/4fa12de0ede3910ce805ccbb849cfd89fd003ba5